lib/ruby_lsp/requests.rb

```diff
@@ -40,6 +40,7 @@ module Requests
     autoload :DocumentHighlight, "ruby_lsp/requests/document_highlight"
     autoload :InlayHints, "ruby_lsp/requests/inlay_hints"
     autoload :Completion, "ruby_lsp/requests/completion"
+    autoload :CompletionResolve, "ruby_lsp/requests/completion_resolve"
```

Comment:
Let's add this to the list of comments above right below completion, so that it shows up in the docs.